### ISSUE TITLE
As a restaurant owner, I want to export the menu

### ISSUE THREAD

**Issue by AZhiKai** [CS2103-AY1819S1-F10-4/main#19]
so that it can be used for many purposes such as consulting with a friend on what can be improved, or archiving it in a backup drive.

**Comment by AZhiKai** [comment]
Medium. Not sure how useful this would be.

**Comment by m4dkip** [comment]
Medium

**Comment by AZhiKai** [comment]
Agree. According to our TA's latest email, he say we should focus on our target audience being the restaurant owner and that for customers, we can have a export function for the restaurant owner to export the menu so he can make hard copies, for example!

I changed my vote to High.

**Comment by HyperionNKJ** [comment]
High. Would appreciate the convenience if I were the restaurant owner

**Comment by AZhiKai** [comment]
Depends on how we all handle storing the data, we might very well be able to export all kinds of data.

More to be said during our meeting. 😄

**Comment by AZhiKai** [comment]
We probably won't have the time to do import/export. Perhaps whoever no longer has any features for v1.4 can consider taking up this task of exporting and importing xml.
